src/cli/task-help.ts

```diff
@@ -1,12 +1,16 @@
-import type { CompilerSystem, Logger } from '../declarations';
+import type { Config } from '../declarations';
+import { taskTelemetry } from './task-telemetry';
 
-export const taskHelp = (sys: CompilerSystem, logger: Logger) => {
-  const p = logger.dim(sys.details.platform === 'windows' ? '>' : '$');
+export const taskHelp = async (config: Config) => {
```

Comment:
Is there a need to pass `Config` as an argument here, could we instead call `getCompilerSystem()` and `getLogger()` on `stencil-cli-config.ts`? What's the preferred interface to use for this singleton?

Reply:
I would love to migrate to use the singleton in more cases like what you point out - I just didn't want to change too much. I personally enjoy the ergonomics of function calls without any props, as you can tell from the code I added. 

For the singleton, my thought process was, as we come in and make modifications, let's refactor code to use the singleton more often. I'm happy to go farther, but want to keep this PR scoped closer to Telemetry than general cleanup and ergonomics.

Reply:
See https://github.com/ionic-team/stencil/pull/2964#discussion_r678323780 - I switched this out.